sa-daily-report.rb

```diff
@@ -57,6 +57,8 @@
 require 'zlib'
 require 'date'
 require 'net/smtp'
+require 'mail'
+require "mail/parsers/content_type_parser"
```

Comment:
Please change this to single quotes.

Reply:
Sorry, I've overseen that. It is already changed in the next commit.